src/applications/mhv-secure-messaging/containers/EditContactList.jsx

```diff
@@ -86,27 +74,31 @@ const EditContactList = () => {
     );
   };
 
-  const handleSaveAndExit = async (e, forceSave) => {
+  const navigateBack = useCallback(
+    () => {
+      if (previousUrl) {
+        history.push(previousUrl);
+      } else {
+        history.push(Paths.INBOX);
+      }
+    },
+    [history, previousUrl],
+  );
+
+  const handleSave = async e => {
     e.preventDefault();
     if (!isMinimumSelected) {
       await setCheckboxError(ErrorMessages.ContactList.MINIMUM_SELECTION);
       focusOnErrorField();
     } else {
-      if (forceSave) {
-        await setIsNavigationBlocked(false);
-      }
-
-      if (forceSave || !isNavigationBlocked) {
-        dispatch(updateTriageTeamRecipients(allTriageTeams));
-        dispatch(
-          addAlert(
-            ALERT_TYPE_SUCCESS,
-            null,
-            Alerts.Message.SAVE_CONTACT_LIST_SUCCESS,
-          ),
-        );
-      }
-      navigateBack();
+      await dispatch(updateTriageTeamRecipients(allTriageTeams));
+      dispatch(
+        addAlert(
+          ALERT_TYPE_SUCCESS,
+          null,
+          Alerts.Message.SAVE_CONTACT_LIST_SUCCESS,
+        ),
+      );

```

Comment:
should we add this `ALERT_TYPE_SUCCESS` dispatch to `updateTriageTeamRecipients` function in actions? I guess it would make more sense since we are handling dispatching error alert content there too